src/Randomizer.App/Windows/GenerateRomWindow.xaml

```diff
@@ -271,6 +190,11 @@
                 <TextBox x:Name="SeedInput"
                          Text="{Binding Seed}" />
               </controls:LabeledControl>
+
+              <controls:LabeledControl Text="Config string (optional):">
```

Comment:
"Config string" might not be the best name for this in the UI... Maybe import/export buttons? Or "Copy settings" and "Import from clipboard" or something like that?